.github/workflows/rust.yml

```diff
@@ -0,0 +1,23 @@
+name: Rust
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Build
+      run: cargo build --verbose --manifest-path=rs/Cargo.toml
```

Comment:
Can this build execute any arbitrary code from the files being checked in? E.g. if it was package.json - it could run scripts, and somebody could add malicious code and try to run it in the PR workflow. 

If it can, please configure the workflow to only run in PRs from trusted users.

Reply:
It's a setting on the repo, not on individual workflows.

IDK if you can see it in the repo settings, but here is what it looks like for me:

![image](https://github.com/microsoft/dev-tunnels/assets/13093042/9ebc984f-4147-4847-b1e0-24dce0e6ef9c)

Reply:
So if any external user submits a PR there will be a button on it that allows only team members to approve the workflows for the PR. It is up to us to review the changes enough to verify there are is no malicious code before approving.

Reply:
This is great, thanks for pointing this out Jason! I think in that case this doesn't have to do with this specific PR and instead is set for the entire repo, so I will go ahead and resolve this comment